packages/paste-website/src/components/icons-list/IconsList.tsx

```diff
@@ -16,7 +16,6 @@ import type {IconObject, IconComponent, IconsListProps, GroupedList} from './typ
 import {IconCard} from './IconCard';
 import {SiteLink} from '../SiteLink';
 import {IconListItem} from './IconListItem';
-import {STICKY_COLUMN_OFFSET} from '../../constants';
```

Comment:
suggestion: can you delete this constant from the constants file as well? Doesn't seem to be used elsewhere